lib/sms_sender.rb

```diff
@@ -0,0 +1,32 @@
+require 'twilio-ruby'
+require 'time'
+require 'dotenv'
+
+Dotenv.load 'config.env'
+
+class SmsSender
```

Comment:
I notice that this class doesn't have any tests 😱 

I'm guessing this is because running the tests was sending you an sms every time? Did Sophie discuss how to properly isolate tests for Twilio in your code review session on Monday? If so, you might want to have a go at implementing this.